nibabies/workflows/bold/alignment.py

```diff
@@ -166,7 +168,8 @@ def init_subcortical_mni_alignment_wf(*, name='subcortical_mni_alignment_wf'):
     workflow.connect([
         (inputnode, applyxfm_atlas, [
             ("bold_file", "in_file"),
-            ("atlas_roi", "reference")]),
+            ("atlas_roi", "reference"),
+            ("atlas_xfm", "in_matrix_file")]),
```

Comment:
I think it should also have gone from `fsl.FLIRT` to `fsl.ApplyXFM`.

Reply:
whoops, nice catch - fixed in https://github.com/nipreps/nibabies/commit/aaf4f069578d678fa76b2274ec05280edadb45af